imports/blocks/give/fieldsets/confirm/index.js

```diff
@@ -71,22 +76,25 @@ export default class Confirm extends Component {
     );
   }
 
-  choose = (e) => {
+  choose = (e: Event) => {
```

Comment:
@jbaxleyiii I tried using that at first, and it wasn't very happy. I'm kind of glad though, because I like the added safety check. Now we know that `id` will exist on `target` when that code runs.